src/mono/netcore/System.Private.CoreLib/src/ILLink/ILLink.LinkAttributes.wasm.xml

```diff
@@ -116,12 +110,52 @@
       <attribute internal="RemoveAttributeInstances" />
     </type>
 
-    <!-- Microsoft.ComponentModel -->
+    <!-- System.ComponentModel -->
     <type fullname="System.ComponentModel.EditorBrowsableAttribute">
       <attribute internal="RemoveAttributeInstances" />
     </type>
   </assembly>
 
+  <!-- The following attributes are only necessary when debugging is supported -->
+  <assembly fullname="System.Private.CoreLib" feature="System.Diagnostics.Debugger.IsSupported" featurevalue="false">
```

Comment:
As all these are feature-switch covered settings which are not wasm specific and we should move them to shared SPC `ILLink.LinkAttributes.xml`

Reply:
I'm not sure I understand what makes the feature-switch covered attribute different from the rest of the attributes in this file. Why aren't the rest of them in the shared SPC?

Reply:
The default SDK setup is trim with copy so it feels odd to include the list of always removed attribute for everyone but I'm not against ;-)